### DISCUSSION TITLE
Search & Select channels - totaly empty

### DISCUSSION THREAD

**Discussion by krzakx** [danielhrisca/asammdf#843] [Q&A]
Hi! 
I do need to analyse some of .dat files however it is very uncomfortable because my _Search tools window_ is totally empty. Even if I will select whole channels on _Channels_ tab.

Any suggestion for me please?
After exporting into csv - same, completely no any channels inside _Search tool window_

**Comment by danielhrisca** [comment]
Please share some screenshot, it's not clear what is going on

**Comment by krzakx** [comment]
Search window/tool is not working for me.
I am trying to work on .dat file.
Even if I type real channel name into search field - there is no search result.

![image](https://user-images.githubusercontent.com/18627339/228856516-1d35e262-d374-4e04-809c-693a636f1f15.png)

**Reply by danielhrisca** [reply]
Use * in front and at the end

**Comment by krzakx** [comment]
Oh.... of course I've been trying to use wildcard
![image](https://user-images.githubusercontent.com/18627339/228888302-37966ce2-ae13-42c9-a857-13a4e3478f2d.png)

**Reply by danielhrisca** [reply]
please try the development branch code
